### ISSUE TITLE
Penicillin allergy

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6416]
**Preferred term label:**
Penicillin allergy

**Synonyms**
Allergy to penicillin

**Definition (free text, please give PubMed ID)**
Hypersensitivity in form of an adverse immune reaction against penicillin.  PMID: 31319528

**Parent term (use hpo.jax.org/app)**
Drug allergy

**Comment by pnrobinson** [comment]
@johannaest  I think that it would be better to store the information about the type of drug to which there is an allergy separately, because otherwise we would wind up with one HPO term <X> allergy for each drug X. We would make an exception to this if allergy against a specific medication could be used as a differential diagnostic sign, but penicillin allergies are among the most common ones.
